test/integration/crud/misc_cursors.test.js

```diff
@@ -3047,37 +2370,27 @@ describe('Cursor', function () {
       }
 
       const configuration = this.configuration;
-      client.connect((err, client) => {
-        expect(err).to.not.exist;
-        this.defer(() => client.close());
+      await client.connect();
```

Comment:
same with this test.  isn't this a duplicate

Reply:
Duplicate, removing.